scripts/rpm/preuninstall.in

```diff
@@ -14,5 +14,21 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+set -e
 
 /usr/bin/falco-driver-loader --clean
+
+# pre uninstall mirrored from .deb
+if [ -d /run/systemd/system ] && [ $1 -eq 0 ]; then
```

Comment:
We should stop falco.service before, otherwise `falco-driver-loader --clean` will bother that it can't `rmmod falco` module (because it is being used by falco itself).

Reply:
👍 Moved the service-stop if-block above the driver cleanup in rpm script